django_rq/urls.py

```diff
@@ -30,4 +30,7 @@
     re_path(
         r'^queues/(?P<queue_index>[\d]+)/(?P<job_id>[-\w\.\:\$]+)/enqueue/$', views.enqueue_job, name='rq_enqueue_job'
     ),
+    re_path(
+        r'^queues/(?P<queue_index>[\d]+)/(?P<job_id>[-\w\.\:\$]+)/stop/$', views.stop_job, name='rq_stop_job'
```

Comment:
Mind updating the regex for `job_id` to match the newly updated one in this PR? https://github.com/rq/django-rq/pull/586/files

Reply:
Done.